website/content/docs/commands/ui.mdx

```diff
@@ -48,3 +59,10 @@ Open the UI directly to look at an allocation:
 $ nomad ui d4005969
 Opening URL "http://127.0.0.1:4646/ui/allocations/d4005969-b16f-10eb-4fe1-a5374986083d"
 ```
+
+Open the UI and login using your ACL token:
+
+```shell-session
+$ NOMAD_ACL_TOKEN=e9674b26-763b-4637-a28f-0df95c53cdda nomad ui -login
+Opening URL "http://127.0.0.1:4646"
+```
```

Comment:
It looks to me that in the [code that opens the URL](https://github.com/hashicorp/nomad/blob/85ed8ddd4fc41b371fb4d652bb9149129893ae6d/command/ui.go#L162-L166) it would include the query parameter in the “Opening URL …” message but from an outside in sense I think leaving out the query parameter would be preferable 🤓 so I support this being the interface 😀

What do you think about noting the presence of `-login` somehow, like “Opening URL "http://127.0.0.1:4646" with token”… maybe unnecessary 🤔

Reply:
That's a good point... I don't think we want to put the one-time token query param in there but giving them some kind of notice that we are passing the token sounds like a great idea. Will fix.

Reply:
Done in 9f22b99